src/components/Nav/SearchBar.jsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import styled from 'styled-components';
+import theme from '../../styles/theme';
+import { fontMix } from '../../styles/mixin';
+
+const SearchBar = () => {
+  return (
+    <SearchBarBackground>
+      <SearchBarContainer>
+        <SearchBarWrap>
+          <LocationButton>장소</LocationButton>
+          <CheckInButton>체크 인 날짜</CheckInButton>
+          <CheckoutButton>체크 아웃 날짜</CheckoutButton>
+          <SearchButton>Search</SearchButton>
+        </SearchBarWrap>
+      </SearchBarContainer>
+    </SearchBarBackground>
+  );
+};
+
+const SearchBarBackground = styled.div`
+  display: flex;
+  align-items: center;
+  width: 100%;
+  height: 60px;
+  background-color: ${theme.mainYellow};
```

Comment:
```suggestion
  background-color: ${props => props.theme.mainYellow};
```

theme은 이미 index.js에서 props로 넘겨주고 있어서
각 컴포넌트에서 import 하지 않으셔도 됩니다!
theme에 있는 스타일을 사용할 때 위와 같은 방법을 적용해주세요